tests/integration/security/policy_attachment_only/jwt_gateway_test.go

```diff
@@ -190,6 +190,86 @@ func TestGatewayAPIRequestAuthentication(t *testing.T) {
 		})
 }
 
+func TestGatewayAPIAuthorizationPolicy(t *testing.T) {
+	framework.NewTest(t).
+		Label(label.IPv4). // https://github.com/istio/istio/issues/35835
+		Features("security.authorization.gateway-api").
+		Run(func(t framework.TestContext) {
+			crd.DeployGatewayAPIOrSkip(t)
+			config.New(t).
```

Comment:
Are the configs being built and applied correctly? Are these params correct?

Reply:
Yes the building is set up correctly. I believe the error lies with the rules in my authorization policy config. I have modified it.